model/task/task.go

```diff
@@ -106,13 +106,15 @@ type Task struct {
 	// Tags that describe the task
 	Tags []string `bson:"tags,omitempty" json:"tags,omitempty"`
 
-	// The host the task was run on. This value is empty for display
-	// tasks
+	// The host the task was run on. This value is empty for display tasks
 	HostId string `bson:"host_id" json:"host_id"`
 
 	// The version of the agent this task was run on.
 	AgentVersion string `bson:"agent_version,omitempty" json:"agent_version,omitempty"`
 
+	// Set to true if the task should be concerned for mainline github checks
```

Comment:
Do you mean considered instead of concerned?

Reply:
yes 😅